test/e2e/configuration_test.go

```diff
@@ -73,6 +81,64 @@ func TestPodResourceConfiguration(t *testing.T) {
 				t.Errorf("expected memory limit of 512Mi, found: %s", mem.String())
 			}
 		}
+
+	}
+}
+
+func TestPodTolerationsConfiguration(t *testing.T) {
+	client := framework.MustNewClientset(t, nil)
+
+	defer framework.MustRemoveImageRegistry(t, client)
+
+	tolerations := []corev1.Toleration{
+		{
+			Key:      "mykey",
+			Value:    "myvalue",
+			Effect:   "NoSchedule",
+			Operator: "Equal",
+		},
+	}
+
+	cr := &imageregistryapiv1.Config{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: imageregistryapiv1.SchemeGroupVersion.String(),
+			Kind:       "Config",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: imageregistryapiv1.ImageRegistryResourceName,
+		},
+		Spec: imageregistryapiv1.ImageRegistrySpec{
+			ManagementState: operatorapiv1.Managed,
+			Storage: imageregistryapiv1.ImageRegistryConfigStorage{
+				Filesystem: &imageregistryapiv1.ImageRegistryConfigStorageFilesystem{
+					VolumeSource: corev1.VolumeSource{
+						EmptyDir: &corev1.EmptyDirVolumeSource{},
+					},
+				},
+			},
+			Replicas:    1,
+			Tolerations: tolerations,
+		},
+	}
+	framework.MustDeployImageRegistry(t, client, cr)
+	framework.MustEnsureImageRegistryIsAvailable(t, client)
+	framework.MustEnsureClusterOperatorStatusIsSet(t, client)
+
+	deployments, err := client.Deployments(imageregistryapiv1.ImageRegistryOperatorNamespace).List(metav1.ListOptions{})
```

Comment:
The deployment name is `image-registry`, so you can use Get.

Reply:
updated in current push